aten/src/ATen/native/native_functions.yaml

```diff
@@ -8034,6 +8034,14 @@
     MPS: random_mps_
   autogen: random.from, random.from_out
 
+- func: random_.from_int(Tensor(a!) self, int from, int? to, *, Generator? generator=None) -> Tensor(a!)
+  device_check: NoCheck   # TensorIterator
+  variants: method
+  tags: nondeterministic_seeded
+  autogen: random.from_int, random.from_int_out
+  dispatch:
+    CompositeImplicitAutograd: random_
```

Comment:
yep I think this is fine (we have aliases for e.g. `aten.subtract -> aten.sub`, that are also implemented by just added a CompositeImplicitAutograd entry for `subtract`).

You'll still need to write the actual kernel though that calls from `random_ -> old_fn`